apps/dashboard/config/configuration_singleton.rb

```diff
@@ -315,6 +315,21 @@ def can_access_file_editor?
     can_access_core_app? 'file-editor'
   end
 
+  # Maximum file upload size that nginx will allow from clients in bytes
+  #
+  # @example No maximum upload size supplied.
+  #   file_upload_max #=> "10737420000"
+  # @example 20 gigabyte file size upload limit.
+  #   file_upload_max #=> "21474840000"
+  # @return [String] Maximum upload size for nginx.
+  def file_upload_max
+    [ENV['FILE_UPLOAD_MAX']&.to_i, ENV['NGINX_FILE_UPLOAD_MAX']&.to_i].compact.min || 10737420000
+  end
+
+  def allowlist_paths
+    (ENV['ALLOWLIST_PATH'] || ENV['WHITELIST_PATH'] || "").split(':').map{ |s| Pathname.new(s) }
```

Comment:
We should change this to `ALLOWLIST_PATH` just for consistency's sake if nothing else.